apitest/src/test/resources/logback.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What does this file do?

Reply:
The `<root level="XXX">` sets the log.level in the test case classes in `apitest/src/test/java/bisq/apitest`, when you run tests within the IDE.  There are some log.debug statements in the jupiter tests.  This file determines whether I see those log.debug outputs or not.  This file has no effect on any other logging, outside of `apitest/src/test/java/bisq/apitest`.